CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -2,6 +2,11 @@ Note that we start to track changes starting from v3.6.
 
 <hr>
 
+## v3.6.0-alpha.1 (TBD)
+
+### Others
+- [Bump go version to 1.22.6](https://github.com/etcd-io/raft/pull/210)
+
```

Comment:
As mentioned in https://github.com/etcd-io/bbolt/pull/815#issuecomment-2276603973, I tent not to update changelog for golang version bump

Reply:
The PR is updated! :)